src/main.cc

```diff
@@ -1211,22 +1232,19 @@ int main(int argc, char *argv[0])
 
   }
 
-  // Runmode: 0 for all, 1 for clustersize, 2 for threshold, 3 for threshold and clustersize, 4 for noise, 5 for E plot, 9 for testing...
-  // 6 for clustersie only, 7 for pointing, 10 for testing of GBLwidths, 11 optimising thickness and Highlanf factor
 
-  Int_t runmode;
 
   // Open config file
-  std::string s_config = "../conf/config.txt";
-  ifstream conf(s_config.c_str());
-  if(!conf)
-  {
-    std::cout << "../conf/config.txt file missing!" << std::endl;
-    return 1;
-  } else
-  {
-    conf >> runmode;
-  }
+  //std::string s_config = "../conf/config.txt";
```

Comment:
instead of commenting out, next better just delete it. The old code is still git-ed, so we dont lose anything.